Kickstarter-iOS/DataSources/SettingsNewslettersDataSource.swift

```diff
@@ -0,0 +1,29 @@
+import Library
+import KsApi
+
+internal final class SettingsNewslettersDataSource: ValueCellDataSource {
+
+  internal enum Section: Int {
+    case newsletters
+  }
+
+  internal func load(newsletters: [Newsletter], user: User) {
+
+    let newsletter = newsletters.enumerated().map { _, newsletter in
```

Comment:
I don't think you need the `enumerated()` here since you're not using the index.

Reply:
Ops